samoa-api/src/main/java/com/yahoo/labs/samoa/evaluation/measures/SilhouetteCoefficient.java

```diff
@@ -60,6 +60,7 @@ public void evaluateClustering(Clustering clustering, Clustering trueClustering,
     for (int p = 0; p < points.size(); p++) {
       DataPoint point = points.get(p);
       ArrayList<Integer> ownClusters = new ArrayList<>();
+      double pointInclusionProbThreshold = 0.8;
```

Comment:
What if we use `private static final double` for this threshold?
So that the threshold is immutable and we don't need unnecessary declaration in line 63.